berkeley-mobile/StudyPact/CreatePreference/ReviewPreferencesViewController.swift

```diff
@@ -50,6 +51,7 @@ class ReviewPreferencesViewController: UIViewController {
     
     override func viewWillAppear(_ animated: Bool) {
         updateLabels()
+        Analytics.logEvent("opened_studypact_reviewPreferences", parameters: nil)
```

Comment:
change to use underscores instead of camelcase (opened_studypact_review_preferences)

Reply:
same for the rest